exercises/baffling-birthdays/instructions.md

```diff
@@ -0,0 +1,15 @@
+# Instructions
+
+Your task is to implement a solution that verifies the Birthday Paradox's probabilities.
+
+To do this, you need to:
+
+- Determine if two birthdates have the same month and day.
+- Generate random birthdates.
+- Check if a set of randomly generated birthdates contains at least one matching pair.
+- Calculate the probability of at least one match for different group sizes.
```

Comment:
"Calculate" sounds like "Find the an analytical solution", which can easily be done for this problem. I think you could use "Estimate" instead if you want .

(and just for fun, let me see if I can find the solution)
The probability of at least one match in N people is 1 minus the probability of no match at all.
The probability to pick a different date N times in a pool of D (365) days is:
$1 * (D-1)/D * (D-2)/D * ... * (D-N+1)/D = D!/N!/D^N$
(first pick is guaranteed, second pick only has D-1 choices, etc)
that's if N <= D, if N > D, the probability is 0.

Reply:
Fair point! I'll update